src/components/forms/CustomizableComponents/CC_NumberInput.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Form, Input, Radio } from 'antd';
+
+function CC_NumberInput({ onCreate, onCancel, visible, programs, categories }) {
+  return (
+    <div>
+      <h3>Custom Field Entry Name</h3>
+      <Input
+        label="Custom Field Entry Name"
+        placeholder="What is the name of your custom field?"
+        maxLength="150"
+      />
+      <br></br>
+      <h3>Is the Service Provider required to fill out this field?</h3>
+      <Radio>Required</Radio>
+      <br></br>
+      <h3>Custom Field Entry Instructions</h3>
+      <Input
+        placeholder="What instructions does your Service Provider need to know?"
+        maxLength="400"
+      />
+    </div>
+    // <Form.Item
```

Comment:
I think this commented out code will definitely help new developers understand the direction in where you were headed. However, maybe add a commented out line attached to it to explain to them what exactly is going to be rendered by it? Just a thought :)